libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1986,7 +1986,9 @@ void TypeChecker::endVisit(Literal const& _literal)
 			m_errorReporter.warning(
 				_literal.location(),
 				"This looks like an address but has an invalid checksum. "
-				"If this is not used as an address, please prepend '00'."
+				"If this is not used as an address, please prepend '00'. "
+				"Correct checksummed address: '" + _literal.getChecksumAddress() + "'. "
+				"For more information, please see https://github.com/ethereum/EIPs/issues/55"
```

Comment:
I think this should instead point to https://solidity.readthedocs.io/en/develop/types.html#address-literals

Reply:
Gotcha. I'll change that.